src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -256,26 +255,36 @@ def run(self):
         ), hermetic_environment_as(
             **self._env
         ), encapsulated_global_logger():
+
+            exit_code = PANTS_SUCCEEDED_EXIT_CODE
             try:
                 # Clean global state.
                 clean_global_runtime_state(reset_subsystem=True)
 
-                options, _, options_bootstrapper = LocalPantsRunner.parse_options(
+                options, build_root, options_bootstrapper = LocalPantsRunner.parse_options(
```

Comment:
It looks like `build_root` is never used? If so, consider using `_build_root` or sticking with `_` so that we know it's not used.

Reply:
Ah, yeah, I was going to actually use this in some other part of the code but that's moot now, will change.